### ISSUE TITLE
Problem with "bolded" math formulas

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-nist#120]
This code from SP 800-90B:

```adoc
Three unkeyed functions have been vetted for an unkeyed conditioning component:

. Any *approved* hash function specified in FIPS 180 or FIPS 202,

. *stem:[text(Hash)_df]*, as specified in SP 800-90A, using any *approved* hash function specified in FIPS 180 or FIPS 202, and

. *stem:[text(Block_Cipher)_df]*, as specified in SP800-90A using the AES block cipher (see FIPS 197).
```

Generates this:

<img width="717" alt="Screen Shot 2019-05-10 at 5 13 02 PM" src="https://user-images.githubusercontent.com/11865/57516214-df229000-7346-11e9-8308-9d3933613d80.png">

Which is wrong -- "-O--" is clearly not the math formula given.

Immediately after this text, there is this from a table:
```adoc
| stem:[text(Hash)_df] | hash-function output size | hash-function output size
| stem:[text(Block_Cipher)_df] | AES key size | AES key size
```

And this is rendered correctly as:
<img width="811" alt="Screen Shot 2019-05-10 at 5 14 50 PM" src="https://user-images.githubusercontent.com/11865/57516351-20b33b00-7347-11e9-8442-9fca81daa22c.png">

This shows that the `*{...}*` bolding does funny things with a Math formula. If a Math formula cannot be bolded, we should skip its processing.

-------

And after I remove the `*{...}*` from the text, i.e.
```adoc
. *stem:[text(Hash)_df]*, as specified in SP 800-90A, using any *approved* hash function specified in FIPS 180 or FIPS 202, and
```

It works properly:

<img width="713" alt="Screen Shot 2019-05-10 at 5 16 14 PM" src="https://user-images.githubusercontent.com/11865/57516452-535d3380-7347-11e9-857e-36ad50210bac.png">

**Comment by opoudjis** [comment]
This is happening because metanorma-standoc is still refusing to consider any formatting within an inline formatting tag. 

What's puzzling to me is that I thought I'd already relaxed that constraint. Investigating.

**Comment by opoudjis** [comment]
Yes, https://github.com/metanorma/metanorma-ogc/issues/10; but the solution was `<< node.text`, not `<< node.content`. That was dealing with mixing bold and italic (so long as the nested formatting was the double-delimiter, word-part formatting, e.g. `*__a__*`, but not dealing with this. Using `node.content`.

**Comment by opoudjis** [comment]
We are running into how Asciidoctor handles inline formatting. It handles it by substitution: 

* Inline formatting is treated as a "Quote": that includes not just quotation marks, but also bold, italics, subscript, superscript
* Any "quotes" within "quotes" are initially left alone
* When the quote substitution stage happens, quotes within quotes are substituted by the rendering of that quote

So `*__a__*` is initially parsed as `<strong>__a__</strong>`, and is then postprocessed by substitution as `<strong><em>a</em></strong>`. Using a regex: `QuotedTextSniffRx = { false => /[*_`#^~]/, true => /[*'_+#^~]/ }`

This of course is going to simply mangle any XML markup that has ended up in there; the XML markup `<stem>...</stem>`, embedded within `<strong>...</strong>`, is ending mangled as \u00961\u0097 (which are in fact trailing garbage characters after the end of string).

I cannot replicate this with other macros: `*link:url[text]*`is passed through fine. I do not know why the stem inline macro is in effect stripped out if it is embedded in inline markup, before the substitution step gets to it. This issue has not been fixed in Asciidoctor 2.0.

All that can be done is to get rid of any boldfaced maths in documents, as something Asciidoctor does not allow us to deal with properly.
